### PR TITLE
[FIX] web_editor: fix pictograms in rte

### PR DESCRIPTION
**Pull request by dmo-odoo**
This should fix https://github.com/odoo/odoo/issues/44689

### CODE REVIEW

**Comment by qsm-odoo** [comment]
@dmo-odoo Probably ok for master but in stable we cannot just remove modules and functions... maybe replace the duplicated functions with a reference to the actual function by important the fonts module in the base module would be possible? Not sure.

**Comment by dmo-odoo** [comment]
@qsm-odoo You're right. It's been a long time since I worked on a stable branch... ;-) I minimized the impact of the fix in this PR and opened a master one: https://github.com/odoo/odoo/pull/46271

**Comment by qsm-odoo** [comment]
:+1: If it works, fine by me :)

File: addons/web_editor/static/src/js/editor/rte.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 odoo.define('web_editor.rte', function (require) {
 'use strict';
 
-var base = require('web_editor.base');
+var base = require('wysiwyg.fonts');
```
Comment (Gorash) [inline addons/web_editor/static/src/js/editor/rte.js]: base ?

**Comment by dmo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**